pkg/nuctl/command/export.go

```diff
@@ -23,8 +23,10 @@ func newExportCommandeer(rootCommandeer *RootCommandeer) *exportCommandeer {
 	}
 
 	cmd := &cobra.Command{
-		Use:   "export",
-		Short: "Export resource to json format",
+        Use:   "export",
+		Short: "Export functions or projects",
+		Long:  "Export the configuration of a specific function or project or of all functions/projects (default)
```

Comment:
```suggestion
		Use:   "export",
		Short: "Export functions or projects",
		Long:  "Export the configuration of a specific function or project or of all functions/projects (default)
```
Inconsistent indentation here, please align the `Use` member to the rest of the members - lint issue